cmd/brew-test-bot.rb

```diff
@@ -533,12 +533,17 @@ def download
           "git", "-C", @tap.path.to_s,
                  "log", "-1", "--format=%h (%s)", "origin/master"
         ).strip
-        puts "#{@tap} origin/master #{tap_origin_master_revision}"
         tap_revision = Utils.popen_read(
           "git", "-C", @tap.path.to_s,
                  "log", "-1", "--format=%h (%s)"
         ).strip
-        puts "#{@tap} HEAD #{tap_revision}"
+        puts <<~EOS
+          Testing tap #{@tap} with:
+            - origin/master   - #{tap_origin_master_revision.blank? ? "(undefined)" : tap_origin_master_revision}
+            - HEAD            - #{tap_revision.blank? ? "(undefined)" : tap_revision}
+            - diff_start_sha1 - #{diff_start_sha1.blank? ? "(undefined)" : diff_start_sha1}
+            - diff_end_sha1   - #{diff_end_sha1.blank? ? "(undefined)" : diff_end_sha1}
```

Comment:
Minor nit: I'm inclined to remove all the eight hyphens. Otherwise looks great! Thank you for this work, Ladislas!

Reply:
Of course! Do I keep the indentation and alignment?

Reply:
@sjackman and also a new line before `Testing tap` to make it clearer in the log?

Reply:
> Do I keep the indentation and alignment?

Yes keep the indentation and alignment, and a new line before `Testing tap` sounds good.